### ISSUE TITLE
Shadow grab bugs

### ISSUE THREAD

**Issue by Vagabond** [omf2097/openomf#253]
2 big bugs with this one:
- The shadow on the way down does not change to the grab quick enough
- Does not immobilize the opponent

There's also issues when the opponent is at the extreme edge of the screen, the shadow never appears. Also the 'grab' duration is waaay too short.

**Comment by Vagabond** [comment]
I fixed the collision detection so this move can at least 'hit' now, but the part where the opponent is immobilized still doesn't happen. I can't actually find an animation that would control that, maybe it is one of the mystery strings in master.dat?

**Comment by Vagabond** [comment]
The shadow grab coming down on the edge of the screen gets culled because I think we kill projectiles that hit the wall, by default?

**Comment by katajakasa** [comment]
Yup, projectiles are set to the finishing animation (if any) and destroyed when they hit the walls.

**Comment by Vagabond** [comment]
I fixed this a couple months ago.
